packages/core/js-sdk/src/client.ts

```diff
@@ -125,59 +152,92 @@ export class Client {
       )
 
       // Any non-request errors (eg. invalid JSON in the response) will be thrown as-is.
-      return fetch(normalizedInput, normalizeRequest(init, headers)).then(
-        (resp) => {
-          this.logger.debug(`Received response with status ${resp.status}\n`)
-          return normalizeResponse(resp, headers)
-        }
-      )
+      return fetch(
+        normalizedInput,
+        normalizeRequest(init, headers, this.config)
+      ).then((resp) => {
+        this.logger.debug(`Received response with status ${resp.status}\n`)
```

Comment:
Q: should we keep that?

Reply:
@adrien2p yup, if you can see how `debug` is defined, if you set `debug` in the config it will log these, otherwise it won't